Filed/AAV1ATE.md

```diff
@@ -0,0 +1,17 @@
+An acronym for remembering all the FAA-required maintenance operations
+
+***Meaning*** :: <u>A</u>nnual, <u>V</u>OR, <u>1</u>00 Hour, <u>A</u>ltimeter, <u>T</u>ransponder, <u>E</u>LT
```

Comment:
Minor: good to add "<u>A</u>D" so it shows up with the other acronyms

Reply:
Will do in the next PR